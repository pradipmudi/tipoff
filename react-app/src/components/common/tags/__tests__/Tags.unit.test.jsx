import React from "react";
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

import TestTags from '../Tags';

describe("Tags", () => {
    const tags=["tag-1", "tag-2","tag-3"];

    it("matches snapshot", () => {
        const tree = renderer.create(<TestTags tags={tags} />).toJSON();
        expect(tree).toMatchSnapshot();
    
    });
    
    it("renders without issues", () =>{
        const component = shallow(<TestTags tags={tags} />);
        expect(component.find('.dark-tag').length).toEqual(tags.length);
        
    });

});

