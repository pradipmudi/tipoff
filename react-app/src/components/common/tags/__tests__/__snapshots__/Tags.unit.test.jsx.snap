// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tags matches snapshot 1`] = `
<div>
  <span>
    <span
      className="tag dark-tag"
      style={
        Object {
          "margin": "5px",
        }
      }
    >
      tag-1
    </span>
    <span
      className="tag dark-tag"
      style={
        Object {
          "margin": "5px",
        }
      }
    >
      tag-2
    </span>
    <span
      className="tag dark-tag"
      style={
        Object {
          "margin": "5px",
        }
      }
    >
      tag-3
    </span>
  </span>
</div>
`;
